test/end-to-end/pages/JudicialSittingRecordsPage.js

```diff
@@ -0,0 +1,8 @@
+const { I } = inject();
+
+function clickAddSittingRecords() {
+  I.see('Judicial sitting records');
+  I.click('Add Sitting Record(s)');
```

Comment:
An explicit wait is needed at the end of this as well, to make sure the next page is loaded.

Reply:
We need a wait on line 6, to make sure that the next page is loaded.